### PR TITLE
Default ResourceManager

### PR DESCRIPTION
**Pull request by vyzo**
On top of #1285

### CODE REVIEW

File: defaults.go [hunk 1]
```diff
@@ -85,6 +86,13 @@ var DefaultEnableRelay = func(cfg *Config) error {
 	return cfg.Apply(EnableRelay())
 }
 
+var DefaultResourceManager = func(cfg *Config) error {
+	limiter := rcmgr.NewStaticLimiter(0.125, 128<<20, 1<<30)
```
Comment (marten-seemann) [inline defaults.go:90]: maybe add a comment what this is in human readable units? (128 MB and 1 GB, if I'm not mistaken?)

File: limits.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	circuitv2 "github.com/libp2p/go-libp2p/p2p/protocol/circuitv2/relay"
+	"github.com/libp2p/go-libp2p/p2p/protocol/holepunch"
+	"github.com/libp2p/go-libp2p/p2p/protocol/identify"
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
+
+	rcmgr "github.com/libp2p/go-libp2p-resource-manager"
+)
+
+// SetDefaultServiceLimits sets the default limits for bundled libp2p services
+func SetDefaultServiceLimits(limiter *rcmgr.BasicLimiter) {
```
Comment (marten-seemann) [inline limits.go:15]: Now that we check if the value already exists, this doesn't need to be exported any more.

**Review by marten-seemann — APPROVED** [review]

File: defaults.go (hunk 1 above)
Comment (vyzo) [reply defaults.go:90]: yeah, good point, will do.

File: limits.go (hunk 2 above)
Comment (vyzo) [reply limits.go:15]: I think we should keep it exported so that the user can call it on his own limiter.

File: defaults.go (hunk 1 above)
Comment (vyzo) [reply defaults.go:91]: done.

**Pull request merged**